### PR TITLE
PR react router and category menu

### PR DESCRIPTION
**Pull request by tomekwoojcik**

### CODE REVIEW

File: src/pages/category-shop/category-shop-context.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  breakpointView: boolean;
+  handleClickOpen: VoidFunction;
+  handleClose: VoidFunction;
+  open: boolean;
+}
+
+export const CategoryShopContext = createContext({} as PropsContextModel);
+
+export const CategoryShopProvider = ({ children }: PropsModel) => {
+  const category = useLoaderData() as CategoriesModel[];
```
Comment (liftoff-project) [inline src/pages/category-shop/category-shop-context.tsx]: I think it should be in the plural form ("categories").

File: src/main.tsx [hunk 2]
```diff
@@ -13,22 +13,25 @@ import ProductShop from "./pages/product-shop/product-shop.tsx";
 import { apiGetCategories, CategoriesModel } from "./api/api-get-categories.ts";
 
 export async function loader() {
-  const category : CategoriesModel[] | undefined = await apiGetCategories("https://lopi2-backend-5517f8f04d28.herokuapp.com/api/");
-  return category ;
-} 
+  const category: CategoriesModel[] | undefined = await apiGetCategories(
+    "https://lopi2-backend-5517f8f04d28.herokuapp.com/api/"
```
Comment (liftoff-project) [inline src/main.tsx]: Please move all references to the backend to an environment variable (https://lopi2-backend-5517f8f04d28.herokuapp.com).

File: src/main.tsx [hunk 3]
```diff
@@ -13,22 +13,25 @@ import ProductShop from "./pages/product-shop/product-shop.tsx";
 import { apiGetCategories, CategoriesModel } from "./api/api-get-categories.ts";
 
 export async function loader() {
-  const category : CategoriesModel[] | undefined = await apiGetCategories("https://lopi2-backend-5517f8f04d28.herokuapp.com/api/");
-  return category ;
-} 
+  const category: CategoriesModel[] | undefined = await apiGetCategories(
```
Comment (liftoff-project) [inline src/main.tsx]: plural form here

File: src/main.tsx [hunk 4]
```diff
@@ -13,22 +13,25 @@ import ProductShop from "./pages/product-shop/product-shop.tsx";
...
   {
     path: "/",
     element: <MainPage />,
     errorElement: <ErrorPage />,
     loader: loader,
-    
+
     children: [
       {
         path: "category",
```
Comment (liftoff-project) [inline src/main.tsx]: plural form? as we are loading categories

Comment (tomekwoojcik) [reply src/main.tsx]: ok

**Pull request merged**